crates/swc_ecma_transforms_compat/src/es2017/async_to_generator.rs

```diff
@@ -564,6 +629,39 @@ impl VisitMut for MethodFolder {
     noop_visit_mut_type!();
 
     fn visit_mut_expr(&mut self, expr: &mut Expr) {
+        match expr {
```

Comment:
To visit inner scoped fn body extended MethodFolder. maybe ok? or should it be separate visitor?